### PR TITLE
Migration update command line.

### PR DESCRIPTION
**Pull request by cheme**
@kianenigma, I switch my code to check migration progress to a simple command.
Maybe it can go to your branch and migration PR (pbbly need to merge master ecr-state-update-host first to see something in the diff).
'check-migration' command should be use.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -569,7 +574,6 @@ cargo-check-macos:
         --substrate
         "$DEPENDENT_REPO"
         "$GITHUB_PR_TOKEN"
-    - cd "$DEPENDENT_REPO" && git rev-parse --abbrev-ref HEAD
```
Comment (TriplEight) [inline .gitlab-ci.yml:572]: All the changes from this file are already in [master](https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml). Dirty rebase?

Comment (cheme) [reply .gitlab-ci.yml:572]: I think it is because kiz-inner-hashing-runtime-migration is not merged with master (this PR is for this other PR).

Comment (TriplEight) [reply .gitlab-ci.yml:572]: oh gotcha

**Review by TriplEight — APPROVED** [review]

**Comment by kianenigma** [comment]
Do we still need this @cheme?

**Comment by cheme** [comment]
No, maybe at some point it may be useful to do quick test, but probably not worth merging in master.

**Pull request closed without merging**